hack/test-cmd.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_C
 os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
 # breaking the config fails the validation check
 cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
-os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+os::util::sed '12,17d' ${BASETMPDIR}/master-config-broken.yaml
```

Comment:
i think it's trying to test the validation with fields missing. yaml deserialization errors don't make the test pass. but it's not a bad idea to add a duplicate with fields missing rather than trying to hack up the existing file. i'll play with it.